apps/cookbook/src/app/examples/chart-example/examples/chart-example-activity-gauge.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component } from '@angular/core';
+
+import { Example } from '../../example.interface';
+
+const config = {
+  selector: 'cookbook-chart-example-activity-gauge',
+  template: `
+    <kirby-card>
+      <kirby-card-header [title]="'Activity-Gauge'"> </kirby-card-header>
+      <div
+        style="background: url('https://images.unsplash.com/photo-1512917774080-9991f1c4c750?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1950&q=80') no-repeat fixed center;"
+      >
+        <kirby-chart
+          [height]="258"
+          [type]="'activitygauge'"
+          [showDataLabels]="false"
```

Comment:
```suggestion
          showDataLabels="false"
```

Reply:
![image](https://user-images.githubusercontent.com/9210691/81777690-ad13b300-94f1-11ea-8e2b-9abbaae7a251.png)
That is not right, as this will convert it to a string